symphony-bdk-config/src/main/java/com/symphony/bdk/core/config/exception/BdkConfigFormatException.java

```diff
@@ -0,0 +1,16 @@
+package com.symphony.bdk.core.config.exception;
+
+import org.apiguardian.api.API;
+
+/**
+ * Thrown a configuration field is not located at the right place in the YAML tree.
```

Comment:
I guess you wanted to say:  "Thrown when a configuration field ..."

Reply:
Fixed